README.md

```diff
@@ -80,10 +80,32 @@ Build with CMake:
        this option is not defined, environment variable LLVM_HOME will be
        used.
 
+     -DLLVMC_FLAGS="additional cflags"        (default : empty)
+       This option offers you the power to pass additional cflags
+       to Clang while building LLVM IR.
+
+     -DTRACER_LD_FLAGS="additional ldflags"   (default : empty)
+       This option offers you the power to pass additional ldflags
+       to g++ while linking instrumented LLVM IR to executable.
+
      -DLLVM_RECOMMEND_VERSION="3.4", "3.5"    (default : 3.4)
        LLVM-Tracer supports both LLVM 3.4 and 3.5. It uses LLVM 3.4 by
        default, but you can manually specify the LLVM version to use.
 
+     -DGCC_INSTALL_PREFIX="/your/gcc/location"       (default : empty)
+       Clang++ requires stdc++ of GCC. This option denotes the
+       path to search libstdc++ and it's headers. If this option
+       is undefined, Clang++ will apply it's default settings.
+       In LLVM auto-installer, this option will be used as the
+       default search path by the installed Clang.
+
+     -DCMAKE_PREFIX_PATH="/your/toolchain/location"  (default : empty)
+       This CMake option denotes the additional path to search
+       dependent tools. In a sitution where you want to use LLVM
```

Comment:
CMAKE_PREFIX_PATH is used to configure the additional search path for any executable or library found by CMake find_package(), find_program() and find_library()
For exmaple, zlib, libboost or python interpreter are found by CMake API.